src/www/index_cordova.html

```diff
@@ -21,6 +21,7 @@
     <title>Outline</title>
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=no" />
     <!-- See https://github.com/webcomponents/custom-elements#es5-vs-es2015 -->
+    <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet" />
```

Comment:
We should bundle these in, right?

Reply:
MWC doesn't bundle it in. I can move it into `third_party`

Reply:
That's probably fine. Notice that you need to so some css for self-hosting: https://google.github.io/material-design-icons/#icon-font-for-the-web

Reply:
Yep, added that to the main styles.css